src/main/java/org/terasology/manualLabor/systems/ShearingSystem.java

```diff
@@ -69,6 +70,7 @@ public void onShearing(AttackEvent event, EntityRef entityRef) {
             delayManager.addPeriodicAction(entityRef, HAIR_REGROWTH_ACTION_ID, 0, HAIR_REGROWTH_TIME / 20);
             switchPrefab(entityRef, SHEARED_SHEEP_MESH, SHEARED_SHEEP_MATERIAL);
         }
+	event.consume();
```

Comment:
whitespaces instead of tab
```suggestion
    event.consume();
```